### PR TITLE
10595 add missing help text to linking suggestions

### PR DESCRIPTION
**Pull request by Dieterrr**
## Summary

This PR can be summarized in the following changelog entry:

* Added a help text to the Internal linking suggestions. 

## Test instructions

This PR can be tested by following these steps:

* Check out this branch and run `yarn link`.
* Check out wordpress-seo-premium and run `yarn link yoast-components`
* Build all the files and run grunt build in the premium folder. 
* Go to a post, look at the linking suggestions. The copy for the help text should be the same as in the [design example](https://user-images.githubusercontent.com/6073772/43400531-3d53cae0-940e-11e8-9798-3a13ed510f97.png). 

## Quality assurance

* [x] I have tested this code to the best of my abilities
* [ ] I have added unittests to verify the code works as intended

Fixes #[10595](https://github.com/Yoast/wordpress-seo/issues/10595)

### CODE REVIEW

File: composites/LinkSuggestions/LinkSuggestions.js [hunk 1]
```diff
@@ -82,15 +82,15 @@ class LinkSuggestions extends React.Component {
 		} );
 
 		let moreCopyMessage = this.props.translate( "Once you add a bit more copy, we'll give you a list of related " +
-			"content here to which you could link in your post." );
+		                                            "content here to which you could link in your post." );
```
[the remark this answers is not in the document]
Comment (Dieterrr) [reply composites/LinkSuggestions/LinkSuggestions.js]: Checked and the spacing seems fine.

**Comment by afercia** [comment]
Acceptance 🚧 Can't fully test because of a few errors after https://github.com/Yoast/wordpress-seo/pull/10519/files

Regardless, there are a couple link inconsistencies:
- one link uses `target="_blank"`, another link doesn't
- I'd suggest to check if these links should open in a new tab
- if so, they should use `makeOutboundLink()` to render the visually hidden a11y message

Minor note about Coding Standards:
Personally, I think mixed tabs and spaces shouldn't ever be used. The indentation for long text strings, typically the ones used for `translate()` uses mixed tabs and spaces for the sake of having aligned lines of text. In these cases, I'd suggest to consider to put the first string chunk in a new line and then use just tabs. This doesn't necessarily need to be addressed in this PR but I'd tend to think we should establish a best practice to follow. /Cc @atimmer

**Comment by afercia** [comment]
Please merge `develop`, as the plugin is expecting the new exports recently added and this PR doesn't include the new exports.

**Comment by atimmer** [comment]
Mixed spaces and tabs are fine as long as tabs are used for indentation and spaces are used for alignment. [This is called smart tabs](https://www.emacswiki.org/emacs/SmartTabs).

I don't necessarily think aligning here is useful. I general I opt for code readability (code is read much more than it is written.) So I would probably go for this variant:

```js
let moreCopyMessage = this.props.translate(
    "Once you add a bit more copy, we'll give you a list of related " +
    "content here to which you could link in your post."
);
```

That both aligns the text so it is easily readable and it doesn't put the text all the way on the right side of the editor.

**Comment by boblinthorst** [comment]
Only minor difference. Is that the comment example ends the first sentence with a `,` , the sentence in this PR ends with a `.`

**Comment by boblinthorst** [comment]
Acceptance. In the design example, the word `Read` is part of the link. in the implementation, it isn't.

**Pull request merged**